packages/inter-protocol/test/psm/gov-add-psm.js

```diff
@@ -0,0 +1,46 @@
+/* global startPSM */
+// @ts-nocheck
+
+/**
+ * @typedef {{
+ *   denom: string,
+ *   keyword?: string,
+ *   proposedName?: string,
+ *   decimalPlaces?: number
+ * }} AnchorOptions
+ */
+
+/** @type {AnchorOptions} */
+const DAI = {
+  keyword: 'DAI',
+  decimalPlaces: 18,
+  denom: 'ibc/toydai',
+  proposedName: 'Maker DAI',
+};
+
+const config = {
+  options: { anchorOptions: DAI },
+  WantStableFeeBP: 1n,
+  GiveStableFeeBP: 3n,
+  MINT_LIMIT: 0n,
+};
+
+/** @param {unknown} permittedPowers see gov-add-psm-permit.json */
+const main = async permittedPowers => {
+  console.log('starting PSM:', DAI);
+  const {
+    consume: { feeMintAccess: _, ...restC },
+    ...restP
+  } = permittedPowers;
+  const noMinting = { consume: restC, ...restP };
+  await Promise.all([
+    startPSM.makeAnchorAsset(noMinting, {
```

Comment:
How can it make a PSM without minting power? all the PSMs will need that

Reply:
This line is only making the anchor asset (issuer, ...). We don't need minting power for that.

Reply:
Ah. I saw it invoking "starPSM" and thought it was starting it